packages/graphback-codegen-input/src/api/ContextTypes.ts

```diff
@@ -70,14 +71,17 @@ export interface InputModelFieldContext {
  * Context type returned from `InputTypeVisitor`
  */
 export interface InputModelTypeContext {
```

Comment:
Naming suggestions:

- TypeContext
- GraphbackModel
- GraphbackType
- InputModel
- GraphbackInputType
- ModelContext
- DataModel

I personally am not a huge fan of prefixing everything with `Graphback` as it just seems like redundant and long without having any extra beneficial information.

What we choose should be applied to all other types also, so `TypeContext`, `FieldContext`, `InterfaceContexr`.

Reply:
Awesome! Would it be possible to get issue and assign that to myself. 
I will action it separately. Do not want to forget about it though